app/config/runtimes.php

```diff
@@ -8,7 +8,13 @@
  */
 $runtimes = new Runtimes();
 
-$allowList = empty(App::getEnv('_APP_FUNCTIONS_RUNTIMES')) ? [] : \explode(',', App::getEnv('_APP_FUNCTIONS_RUNTIMES'));
+$allowString = App::getEnv('_APP_FUNCTIONS_RUNTIMES', 'node-16.0,php-8.0,python-3.9,ruby-3.0,java-16.0');
```

Comment:
I dont think we should hard code these environments here 
They should be fetched from https://github.com/appwrite/appwrite/blob/bac529a270df9c15ac471090722af7ffaad11275/app/init.php#L95

Reply:
You can check it's usage here
https://github.com/appwrite/appwrite/blob/a0daea0bb97ab4bbc483acb374d7c816983da54a/app/tasks/install.php#L39